app/src/configs/env.ts

```diff
@@ -139,4 +139,7 @@ export const MAIN_LOGO_SRC =
 
 export const SUPPORTED_LANGUAGES = (process.env.REACT_APP_SUPPORTED_LANGUAGES ?? '').split(',');
 
-export const ENABLE_LANGUAGE_SWITCHER = process.env.REACT_APP_ENABLE_LANGUAGE_SWITCHER === 'true';
+export const ENABLE_LANGUAGE_SWITCHER =
+  process.env.REACT_APP_ENABLE_LANGUAGE_SWITCHER === 'true' || SUPPORTED_LANGUAGES.length > 0;
```

Comment:
I think this change might result in conflicts..I am presuming a developer consuming `ENABLE_LANGUAGE_SWITCHER` in code will be making their implementations having in mind that `ENABLE_LANGUAGE_SWITCHER` will only be true if its value is set to `true`. Why did we add `|| SUPPORTED_LANGUAGES.length > 0` ?

Reply:
@haseemVD each env variable has a single responsibility here, I would vote we will leave it as it were.

Reply:
Stated in issue
`_Can remove the globe icon if we don't have a language switcher? We need to leave language switcher for projects that require language switching e.g EUSM & Tunisia_`

Then this is already implemented.